pallets/asset-handler/src/lib.rs

```diff
@@ -146,32 +145,33 @@ pub mod pallet {
 		///
 		/// # Parameters
 		///
+		/// * `origin`: `Asset` owner
 		/// * `destination_add`: Recipient's Account
 		/// * `amount`: Amount to be minted in Recipient's Account
 		/// * `rid`: Resource ID
 		#[pallet::weight(T::WeightInfo::mint_asset(1))]
 		pub fn mint_asset(
 			origin: OriginFor<T>,
-			destination_add: [u8;32],
```

Comment:
@serhii-temchenko there is an issue with it. Chainbridge relayer is expecting it to be this.

Reply:
@zktony Can you please point consumer of this in the code? I was thinking that chainbridge is a dependency of asset handler - so asset handler is a consumer of chainbridge and not otherwise.

Reply:
I mean - is it possible to unify this signature on the consumer side so it will send a common type?